packages/cli/src/index.ts

```diff
@@ -106,9 +110,14 @@ program
   .command("sellCompleteSet <marketId> <amount>")
   .option(
     "--seed <string>",
-    "The signer's seed. Default is `//Alice`.",
+    "The signer's seed. Default is:",
     "clean useful exotic shoe day rural hotel pitch manual happy inherit concert"
```

Comment:
```suggestion
    "//Alice"
```

We should change this back to Alice so it works by default on dev chains.